libraries/transformer/src/androidTest/java/androidx/media3/transformer/mh/HdrEditingTest.java

```diff
@@ -139,6 +139,41 @@ public void export_transmuxHlg10File() throws Exception {
     assertThat(actualColorTransfer).isEqualTo(C.COLOR_TRANSFER_HLG);
   }
 
+  @Test
+  public void export_transmuxDolbyVisionFile() throws Exception {
+    Context context = ApplicationProvider.getApplicationContext();
+
+    if (Util.SDK_INT < 24) {
+      // TODO: b/285543404 - Remove suppression once we can transmux H.265/HEVC before API 24.
+      recordTestSkipped(context, testId, /* reason= */ "Can't transmux H.265/HEVC before API 24");
```

Comment:
The below check for format support should check whether muxer supports this format or not so this is not needed.

Reply:
I just did the same thing as export_transmuxHdr10File() and export_transmuxHlg10File(). Shall we remove above code in these three methods together?

Reply:
I think this is not fully resolved as the decoder check (AndroidTestUtil.skipAndLogIfFormatsUnsupported) is still present which should not be required for transmuxing.

Reply:
OK, I'll remove this part of checking.

    if (AndroidTestUtil.skipAndLogIfFormatsUnsupported(
        context,
        testId,
        /* inputFormat= */ MP4_ASSET_DOLBY_VISION_HDR_FORMAT,
        /* outputFormat= */ null)) {
      return;
    }